### PR TITLE
fix minor issue for user mock data

### PR DESCRIPTION
**Pull request by jn1118**
Fixed:
1. Duplicate user id
2. change links to handles for social medias
3. Append data to users instead of user_mock_data

### CODE REVIEW

File: src/firebase/userDb.js [hunk 1]
```diff
@@ -12,13 +12,13 @@ import property_json from "./data/propertyData";
...
 export async function getAllUsers() {
-	const allUsers = "user_mock_data";
-	const querySnapshot = await getDocs(collection(db, allUsers));
-	let users = [];
-	querySnapshot.forEach((doc) => {
-		users.push(doc.data());
-	});
-	return users;
+  const allUsers = "user_mock_data";
+  const querySnapshot = await getDocs(collection(db, allUsers));
```
Comment (bjtrieu) [inline src/firebase/userDb.js]:
```suggestion
  const querySnapshot = await getDocs(collection(db, "users"));
```
this can be changed back to "users" since we're not uploading to "user_mock_data" anymore

File: src/firebase/userDb.js [hunk 2]
```diff
@@ -12,13 +12,13 @@ import property_json from "./data/propertyData";
...
 
 export async function getAllUsers() {
-	const allUsers = "user_mock_data";
-	const querySnapshot = await getDocs(collection(db, allUsers));
-	let users = [];
-	querySnapshot.forEach((doc) => {
-		users.push(doc.data());
-	});
-	return users;
+  const allUsers = "user_mock_data";
```
Comment (aidandenlinger) [inline src/firebase/userDb.js:15]: can this be updated to `users`? means @bjtrieu doesn't need to make another PR just to do this. otherwise looks good, thanks for taking care of this!

**Review by aidandenlinger — CHANGES_REQUESTED** [review]

File: src/firebase/userDb.js (hunk 2 above)
Comment (jn1118) [reply src/firebase/userDb.js]: Fixed!

**Review by aidandenlinger — APPROVED** [review]

**Review by bjtrieu — CHANGES_REQUESTED** [review]
Taking a look at the full document, there are some changes on there I missed before. I'm not sure how to comment on things that haven't been changed so I'll provide it as a comment here:

data/userData.js
- Line 49-51: When something isn't selected or filled out, it's just empty string
- just a little nitpicky, but the contact information doesn't match the user's name for the a couple users, you don't really have to change it if you don't want to but it will look more realistic lol (uid: 12345 , 54321 , 1, def456 (bio doesn't match), 987654321, 123456789, 789012

File: src/firebase/userDb.js (hunk 1 above)
Comment (aidandenlinger) [reply src/firebase/userDb.js]: resolved earlier :+1:

Comment (jn1118) [reply src/firebase/userDb.js]: 1. Changed the "Not Applicable" to empty string 
2. Some of the contact info are randomly generated. I feel like we can leave it like this for now since the contact infos are fake anyway.

Comment (bjtrieu) [reply src/firebase/userDb.js]: sounds good. good to merge on my end!

**Review by bjtrieu — APPROVED** [review]
lgtm
